### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Léon Surbeck

### DISCUSSION THREAD

**Discussion by Bananonymous** [heig-vd-dai-course/heig-vd-dai-course#269] [Show and tell]
https://github.com/Bananonymous

**Comment by ludelafo** [comment]
Very nice profile, thanks! Want kind of games did you develop in JavaScript? :)

**Comment by Bananonymous** [comment]
Thanks !
It was a small "RPG" game that we developed during our CFC with friends "Police Simulator 2000", Top-down view, pixelart, collecting bribes arresting bad guys and so on (not really finished but it was fun)
Also did a "Support" App to help you manage a game of "Werewolves" :)

**Reply by ludelafo** [reply]
Nice projects! The "Police Simulator 2000" looks quite fun haha. Thank you!
